docs/en/operations/settings/settings.md

```diff
@@ -5227,3 +5235,34 @@ Disabled by default.
 The maximum allowed size for String in RowBinary format. It prevents allocating large amount of memory in case of corrupted data. 0 means there is no limit.
 
 Default value: `1GiB`
+
+## S3 settings
```

Comment:
That's not a good place for these settings. This section at the end of this doc is all about Format settings (there is a header `# Format settings`). Let's add these settings before format settings (as setting `engine_file_allow_create_multiple_files`), so we won't mix them up.
BTW, what do you think about transferring format settings into a separate file? In general this doc is really big and almost has no structure (maybe you have ideas how we can improve it).

Reply:
Hmm, do you know why we don't have `Format settings` section in the right section?
<img width="1348" alt="Screenshot 2023-01-23 at 22 55 56" src="https://user-images.githubusercontent.com/48961922/214159015-1746bb20-5f61-4602-b2e2-4712af52788e.png">

Reply:
Maybe the parser gave up after 5,000 lines :(

Reply:
> That's not a good place for these settings. This section at the end of this doc is all about Format settings (there is a header `# Format settings`). Let's add these settings before format settings (as setting `engine_file_allow_create_multiple_files`), so we won't mix them up. BTW, what do you think about transferring format settings into a separate file? In general this doc is really big and almost has no structure (maybe you have ideas how we can improve it).

I will move format settings to another file.

Are there other groups that can move out?

Reply:
AFAIK no other groups. Some time ago format settings were mixed with other settings and I spent some time reorganazing them.

Reply:
About `Format settings` header, just checked other placed in out docs and seems like all headers with single `#` are not presented in the right section.
Do you know how to fix it? Or maybe I should ask someone else

Reply:
> In the `md` file it's `# Format settings {#format-settings}`

Ah, there can only be one level 1 heading in the file, so it got skipped.

Reply:
I am going to have to reorganize this page, there are join settings in here and also in the join docs, merge_tree settings in here and also in the merge_tree docs, etc.  I will come back to this tomorrow.